profile/README.md

```diff
@@ -0,0 +1,64 @@
+and exploring.
+
+This GitHub organization is our attempt to publicize our experiments and thoughts around this
+question. We have no answer, but we hope you'll follow along as we stumble through.
+
+## Tenets
+
+### Define Infrastructure as Code
+
+There is a reason that [the Infrastructure as Code
+movement](https://www.hashicorp.com/resources/what-is-infrastructure-as-code) has gained so much
+popularity in recent years. Ensuring that we define infrastructure in code has many advantages,
+such as:
+
+- It ensures consistency when we provision infrastructure across many environments and regions.
+- It allows the construction of reusable components that define small, single-use pieces of
+  infrastructure, which can then be composed into larger units, much like using libraries in
+  application code.
+- It enables infrastructure provisioning and ongoing management to be automated in the same way as
+  we build and release software.
+- It allows for versioning the definition of your infrastructure, potentially in
+  concert/coordination with the software running on the infrastructure.
+
+For these reasons, we reject any solution that does not allow us to generate, export, or write the
+definition of infrastructure in a form compatible with version-controlled storage.
+
+### Do Not Force Provider Choice
+
+We do not consider solutions that are inherently proprietary or useful only in the context of a
+single cloud provider since this limits their usefulness and leads to the necessity of learning
+multiple languages/frameworks.
+
+### Favor Declarative Over Imperative
+
+We favor tools that follow a declarative versus an imperative approach. Imperative tools require
+users to consider _how to achieve a desired state_, whereas declarative tools only require users to
+_describe the desired result_. For example, imperative tooling requires users to consider factors such
+as idempotence, in-place mutation, and ongoing state management. By contrast, declarative tools
+manage this for you.
+
+### Do Not Require Developers to Learn New Languages
+
+One of our goals is to allow developers to focus on honing their craft rather than learning
+new skills. As such, we favor approaches that enable developers to provision and manage their
+infrastructure either in a language of their choice or using ubiquitously available and used
+languages or formats, such as JSON or YAML.
+
+### Abstract Away Complexity
+
+Managing infrastructure at scale with global redundancy and high availability is a specialized and
+time-consuming skill to achieve and maintain. We seek to offload this task onto specialized
+engineers who focus on these problems while allowing developers to instead focus on achieving their
+applications' maximum performance and availability. As such, we favor solutions enabling the
+complexity of redundant and highly performant infrastructure behind a simple interface that
+developers can understand without knowledge of the underlying infrastructure.
```

Comment:
Nit: Again, I'm a _little_ wary of the us/them division of labor camps described in this paragraph...and I also think there are instances where well-meaning abstraction inadvertently harms efficiency and velocity. So, I kinda question whether abstraction is intrinsically good. More so, I tend to think it's one technique for providing an efficient, effective, and elegant experience.

Reply:
So my intention here was to say, more or less, "We don't want to force folks to understand the nuances of operating infrastructure (and the code that manages it) at scale because that's a complex thing that takes a lot of mental juice and, in many cases, a very specialized set of skills."

For example, operating database platforms at scale across multiple geographic regions is complex and time-consuming. Doing this well takes a relatively deep knowledge of the database platform itself and the cloud provider it runs on. Choosing the right mix of instance types (to balance compute, network, and memory capacity) as well as tuning lower-level parameters to make use of that raw compute best is highly specialized and likely not something a large organization wants to allow every team to make individual decisions on, nor do they want their developers _forced_ into spending their valuable attention capital on acquiring (and maintaining, because cloud providers do change things from time to time) it.

(I emphasized _force_, because here, again, I don't mean to imply that folks _shouldn't_ or _can't_, only that in many cases, it's neither _efficient_ (in terms of time spent and lost productivity on other parts of their job) nor _healthy_ (because of the added stress of expectations and workload) nor _effective_ (because to get this stuff right takes a lot of hard-won experience).)

Is there a better way to say that?